### DISCUSSION TITLE
Custom CSS applies only to index page

### DISCUSSION THREAD

**Discussion by spandl** [observablehq/framework#1744] [Q&A]
I have been following this example to include some custom style settings in a project:
https://github.com/observablehq/framework/tree/main/examples/custom-stylesheet

The setup worked for the index page, but had no impact on all other pages.

**

My goal is to add some minor adjustments (i.e. margins for headlines) to the styles, while keeping all of the layout as is.
What would be the recommended  way of applying these minor updates to the complete site?

Thanks.

**Comment by Fil** [comment]
Can you share your code? If I simply run the example, with `index.md` duplicated as `page2.md`, both pages have the custom styles.

**Reply by spandl** [reply]
I will try to get the code to a GH repo that I can share. Maybe by isolating the issue, I will find out why this happens.
